connector/src/main/java/org/geysermc/connector/entity/PlayerEntity.java

```diff
@@ -264,4 +265,12 @@ public void updateBedrockMetadata(EntityMetadata entityMetadata, GeyserSession s
             }
         }
     }
+
+    /**
+     * Returns the DisplayName if set, otherwise the Username
+     * @return Name of player entity
+     */
+    public String getName() {
+        return displayName==null?username:displayName;
```

Comment:
use whitespace here, to match up with the rest of the Geyser code.

`return displayName == null ? username : displayName`